docs/ops/state/checkpoints.zh.md

```diff
@@ -26,69 +26,49 @@ under the License.
 * toc
 {:toc}
 
-## Overview
+## 概述
+checkpoint 使 Flink 的状态具有良好的容错性，通过 checkpoint 机制，Flink 可以对作业的状态和计算位置进行恢复。
 
-Checkpoints make state in Flink fault tolerant by allowing state and the
-corresponding stream positions to be recovered, thereby giving the application
```

Comment:
In this position,the sentence of "thereby thereby giving the application the same semantics as a failure-free execution" according your suggestion is not  translated and in my first commit it is translated.

Reply:
I think the current translation is enough